README

```diff
@@ -60,13 +70,31 @@ add the required variables to the body section in the client ui
 Click 'SEND'
 
 
-Curl Examples
+Curl Examples:
 If the UI is not your thing, curl probably is! Here's an example of using
 curl to create a gateway node.
 
 curl --insecure --user admin:admin -d "ip_address=192.168.122.104" \
      -X PUT https://192.168.122.69:5000/api/gateway/rh7-gw2
 
+IPv6 Supporting:
+Ceph-iscsi has supported the IPv6 already, for this to make sure the IPv6
+addresses used are global unicast ones, and then the URL link could be:
+
+curl --insecure --user admin:admin -d "ip_address=2006:ac81::1104" \
+     -X PUT https://[2006:ac81::1104]:5000/api/gateway/rh7-gw2
+
+And mix using the IPv4 and IPv6 is also allowed as bellow:
+
+curl --insecure --user admin:admin -d "ip_address=192.168.122.104" \
+     -X PUT https://[2006:ac81::1104]:5000/api/gateway/rh7-gw2
+
+curl --insecure --user admin:admin -d "ip_address=2006:ac81::1104" \
+     -X PUT https://192.168.122.104:5000/api/gateway/rh7-gw2
```

Comment:
Since you changed the example `gwcli` output, can you make the IPs consistent?  In particular, `192.168.122.104` is no longer there.

Reply:
Fixed them all. Thanks.